feature/src/common/picture.h

```diff
@@ -0,0 +1,31 @@
+/**
+ *
+ *  Copyright 2016-2019 Netflix, Inc.
+ *
+ *     Licensed under the Apache License, Version 2.0 (the "License");
+ *     you may not use this file except in compliance with the License.
+ *     You may obtain a copy of the License at
+ *
+ *         http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *     Unless required by applicable law or agreed to in writing, software
+ *     distributed under the License is distributed on an "AS IS" BASIS,
+ *     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *     See the License for the specific language governing permissions and
+ *     limitations under the License.
+ *
+ */
+
+#ifndef PICTURE_H_
+#define PICTURE_H_
+
+typedef struct VmafPicture VmafPicture;
+
+struct VmafPicture
+{
+    float *data_y;
+    float *data_u;
+    float *data_v;
+};
```

Comment:
use `data[3]`. Also, `w`, `h`, `pix_fmt`, and `ptrdiff_t stride[2]` needs to be in here also. This container should be the place to hold other attributes like colorspace, color matrix, etc.

Reply:
This is just a dangling file that I deleted in a local commit. This file will go away.

Reply:
The definition of VmafPicture (what this file used to refer to) is now placed under libvmaf.h